README.md

```diff
@@ -1,105 +1,146 @@
-<p align="center">
-  <a href="https://github.com/actions/typescript-action/actions"><img alt="typescript-action status" src="https://github.com/actions/typescript-action/workflows/build-test/badge.svg"></a>
-</p>
+# @guardian/actions-assume-aws-role
 
-# Create a JavaScript Action using TypeScript
+This is a [GitHub Action][action] for use during [CI].
+Use it to gain temporary credentials to AWS [by exchanging GitHub OIDC credentials for AWS IAM Assume Role][aws-docs] using an `AWS::IAM::OIDCProvider`.
 
-Use this template to bootstrap the creation of a TypeScript action.:rocket:
+## Set up
+[This blog][awsteele] provides a lot of detail.
 
-This template includes compilation support, tests, a validation workflow, publishing, and versioning guidance.  
+In short, you first need to create three resources:
```

Comment:
Is it correct to say that Guardian projects which just want to upload artifacts to Riff-Raff S3 buckets will be able to skip this step? That is, are we all able to share a single policy/role/OIDC provider in the Deploy Tools account?

If so, we might want to make a note of this to avoid confusion.